pkg/internal/collections/collection_helpers_test.go

```diff
@@ -1,6 +1,9 @@
 package collections
 
 import (
+	"errors"
+	"fmt"
+	"github.com/stretchr/testify/assert"
```

Comment:
golangci

Reply:
Yup, I merged with the main just now and ran the pre push